bats/core/api-keys/api-keys.bats

```diff
@@ -67,3 +71,44 @@ new_key_name() {
   error="$(graphql_output '.error.code')"
   [[ "${error}" = "401" ]] || exit 1
 }
+
+@test "api-keys: can create read-only" {
+  key_name="$(new_key_name)"
+
+  variables="{\"input\":{\"name\":\"${key_name}\",\"readOnly\": true}}"
+
+  exec_graphql 'alice' 'api-key-create' "$variables"
+  key="$(graphql_output '.data.apiKeyCreate.apiKey')"
+  secret="$(graphql_output '.data.apiKeyCreate.apiKeySecret')"
+  cache_value "api-key-secret" "$secret"
+
+  readOnly=$(echo "$key" | jq -r '.readOnly')
+  [[ "${readOnly}" = "true" ]] || exit 1
+
+  key_id=$(echo "$key" | jq -r '.id')
+  cache_value "api-key-id" "$key_id"
+
+  exec_graphql 'api-key-secret' 'api-keys'
+
+  name="$(graphql_output '.data.me.apiKeys[-1].name')"
+  [[ "${name}" = "${key_name}" ]] || exit 1
+}
+
+@test "api-keys: read-only key cannot mutate" {
```

Comment:
do we have a test checking that readOnly key can read?

Reply:
Yes in the test above (that creates the read-only key)